ui/app/src/state/store.ts

```diff
@@ -53,7 +55,9 @@ export function createStore(useMock = false): Observable<CrafterCMSStore> {
 }
 
 export function createStoreSync(preloadedState: Partial<GlobalState>): CrafterCMSStore {
-  const epicMiddleware = createEpicMiddleware<StandardAction, StandardAction, GlobalState>();
+  const epicMiddleware = createEpicMiddleware<StandardAction, StandardAction, GlobalState, { intl: IntlShape }>({
+    dependencies: { intl: intlRef.current }
```

Comment:
To take advantaged of the ref, you need to send the whole ref, rather than the value